ui/raidboss/data/07-dt/raid/r3n.txt

```diff
@@ -1,8 +1,84 @@
 ### Arcadion (R3N): AAC Light-heavyweight M3
 
+# 93AB = tp north/mid
+
+# -ii 93CE 93CF 93D2 93D7 9A81 9B44 9B59 9B5B 9B58 9AE0 9AE1 968D 93CD 9B5A 9B5B
+# -it "Brute Bomber"
+
 hideall "--Reset--"
 hideall "--sync--"
 
-0.0 "--Reset--" ActorControl { command: "4000000F" } window 0,100000 jump 0
-
 0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+17.7 "Brutal Impact x4" Ability { id: "93D6", source: "Brute Bomber" }
```

Comment:
Here and elsewhere, do you think it's worth adding a duration?

Reply:
I don't know if the 4 hit lasts long enough to really be worth a `duration`? I'm fine with it either way, whatever makes more sense.